ha/const.py

```diff
@@ -93,6 +95,9 @@
 PCS_CLUSTER_STATUS="pcs cluster status"
 PCS_CLUSTER_UNSTANDBY="pcs cluster unstandby --all"
 PCS_STATUS_NODES="pcs status nodes"
+PCS_NODE_UNSTANDBY="pcs node unstandby <node>"
+PCS_NODE_CLEANUP="pcs resource cleanup --node <node>"
+PCS_NODE_FAILCOUNT_STATUS="pcs resource failcount show --node <node>"
```

Comment:
PCS_FAILCOUNT_STATUS = "pcs resource failcount show"
PCS_NODE_FAILCOUNT_STATUS = PCS_FAILCOUNT_STATUS  + " --node {}"

Above will cover cluster as well as node command.
I am thinking why do you need to check failcount if command is async. Probably we are complicating it.

Reply:
We discussed and changed flow. also updated the code.